app/2.0.x/index.md

```diff
@@ -1,6 +1,7 @@
 ---
-title: Documentation for Kong Gateway
-is_homepage: true
+title: Documentation for {{site.ce_product_name}}
```

Comment:
```suggestion
title: Documentation for Kong Gateway
```
Unfortunately, browser doesn't parse this so the variable is visible in the tab label.